androidsdk/src/test/java/com/wootric/androidsdk/SurveyValidatorTest.java

```diff
@@ -63,6 +63,9 @@ public void checksEligibility_whenNotRecentlySurveyed() throws Exception {
     public void doesNotCheckEligibility_whenRecentlySurveyed() throws Exception {
         User user = testUser();
         EndUser endUser = testEndUser();
+        long createdAtJustNow = new Date().getTime();
```

Comment:
@hachibu we digged deeper into how the `created_at` values are managed on this SDK. @diegoserranoa is right, all date comparisons are in milliseconds (not like the web app) so let's keep it uniform